src/app/campaign-stats.model.ts

```diff
@@ -0,0 +1,6 @@
+export class CampaignStats {
+    constructor(
+        public totalRaisedWithAdjustment: number,
+        public totalCampaignCount: number
```

Comment:
It would be good to have docblocks for these two properties to explain what they mean and what units they're measured in. I don't know if it's worth making the first one into an object that contains the currency code as well as the number, if not then the fact that it's GBP counted in whole pounds should be in the docblock and/or the property name.